tasks/databases.rake

```diff
@@ -311,6 +417,22 @@ namespace :data do
   end
 end
 
+def print_memory_usage
+  memory_before = `ps -o rss= -p #{Process.pid}`.to_i
+  yield
+  memory_after = `ps -o rss= -p #{Process.pid}`.to_i
+
+  puts "Memory usage: #{((memory_after - memory_before) / 1024.0).round(3)} MB"
+end
+
+def print_time_spent
```

Comment:
database.rake is pretty polluted with extra code, in every change i'm trying to get more things out of the rake file. what do you think about moving all this code into it's own .rb file?